src/pages/quiz/Quiz.tsx

```diff
@@ -68,22 +69,39 @@ const QuizPage = () => {
   }, [])
 
   const {
-    settings: { pointValues, scoredResponses, correctAnswers, view },
+    settings: {
+      pointValues,
+      scoredResponses,
+      correctAnswers,
+      view,
+      attemptLimit,
+      timeLimit
+    },
     description,
     title,
     items,
     createdAt,
     updatedAt
-  } = quiz || defaultQuizResponse
+  } = quiz ?? defaultQuizResponse
 
   const points = countPoints(
     items.filter(({ type }) => type !== QuestionTypesEnum.OpenAnswer),
     data
   )
+  const getQuizzes = useCallback(() => {
+    if (cooperationId && quizId) {
+      return ResourceService.getFinishedQuizzesByQuizId(cooperationId, quizId)
+    }
+  }, [cooperationId, quizId])
+
+  const { data: finishedAttempts = [] } = useQuery({
+    queryKey: ['quizzes', cooperationId, quizId],
+    queryFn: getQuizzes
+  })
 
   const addFinishedQuiz = useCallback(() => {
     return ResourceService.addFinishedQuiz({
-      cooperation: id ?? '',
+      cooperation: cooperationId ?? '',
```

Comment:
You have already default value for that
```suggestion
      cooperation: cooperationId,
```